Azordev/dasher-user
# Updated Readme.md with the doc file provided

### Description

I have updated the readme.md with the changes proposed by JHernandez @elpollo12 in her Google Doc file. I have transcribe this to a markdown file and have updated the image to imgur to can be used to the project. I have used HTML to do that the image was be aligned to the left / right with the text

fixes #200 

#### Screenshots

![imagen](https://user-images.githubusercontent.com/66505715/150419850-f69887dc-9edc-44d8-96a1-d4ffe023f468.png)

#### Type of change


### How to test it

1. Go to here to check preview: https://github.com/Azordev/dasher-user/blob/c05e0d5f93caa08a8a57ffa378b8a2ff509dd85e/README.md

### Checklist:

- [x] The .doc file has been transcribed to markdown format
- [x] The Readme.md content now is the content that is at google doc file
- [x] The Readme.md index has been updated to new sections at the file
- [x] The Readme.md preserve the language badges, index and image of project
- [x] The Readme.md have authors and co authors present at the Collaborators section of this project
